config/routes.rb

```diff
@@ -7,6 +7,10 @@
   namespace :admin do
     resources :events
 
+    namespace :event do
+      resources :items
```

Comment:
Why do we need `event/items` resources? Where is it used?

Reply:
@alexshgov To display Event / items in a single dashboard

Reply:
@ZhenyaZhak I expect to see the following url eventually:
```
events/:id/items
```

Reply:
@alexshgov I can not understand what the problem is and what we want to achieve.

Reply:
@ZhenyaZhak there's no need to view all the items. It will be a mess.
Item exists only in the *context of the specific Event*.
```
Event X
   -item_1
   -item_2
   -item_3
```